script.js

```diff
@@ -90,6 +90,11 @@ function highlightElements(elements, className) {
   }, ANIMATION_DURATION);
 }
 
+/**
+ * 
+ * @param {target} event - check if selected word and translation match.
+ */
```

Comment:
Suggestion to first describe the function, then the parameter
```suggestion
 * Check if selected word and translation match.
 * @param {target} event - HTML event object
 */
```